constants.py

```diff
@@ -9,3 +9,5 @@
 HF_EMBEDDINGS_MODEL = 'sentence-transformers/all-MiniLM-L6-v2'
 
 SEVEN_DAYS_IN_SECONDS = 604800
+
+MASADER_GH_REPO = 'ARBML/masader'
```

Comment:
You are using full `GITHUB` in `config.py`, either use the full name here or use the abbreviation there.